docs/src/content/docs/utilities/navigation-end.md

```diff
@@ -0,0 +1,35 @@
+---
+title: navigationEnd
+description: ngxtension/navigation-end
+---
+
+The `navigationEnd` function is a utility for creating an Observable that emits when a navigation ends. It might be used to perform tasks after a route navigation has been completed.
```

Comment:
```suggestion
The `navigationEnd` function is a utility for creating an `Observable` that emits when a navigation ends. It might perform tasks after a route navigation has been completed.
```